### ISSUE TITLE
Add support for React Native 0.44

### ISSUE THREAD

**Issue by silyevsk** [wix/detox#137]
Looks like there are issues with newer React Native versions - a simple test against 0.43 failed because detox found two elements with the same accessibility label (the test passes against 0.40)

**Comment by rotemmiz** [comment]
Added in https://github.com/wix/detox/releases/tag/detox%405.0.9
